examples/flow.cpp

```diff
@@ -171,11 +228,22 @@ try
         return EXIT_FAILURE;
     }
 
-    // Grid init
     std::vector<double> porv = eclipseState->getDoubleGridProperty("PORV")->getData();
-    grid.reset(new GridManager(eclipseState->getEclipseGrid(), porv));
-    auto &cGrid = *grid->c_grid();
-    const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
+#if USE_DUNE_CORNERPOINTGRID
+    // Dune::CpGrid as grid manager
+    typedef Dune::CpGrid  Grid;
+    std::shared_ptr<Grid> gridPtr;
```

Comment:
I just copied what was there. From that I deduce that i went through a review process earlier. Also, creating large stack objects is in general not a good idea.